src/content/structured/components/alerts/code.mdx

```diff
@@ -163,18 +159,21 @@ export const snippetsTitleAbove = [
 export const snippetsResponsiveTitle = [
   {
     language: "Web component",
-    snippet: `<ic-alert heading="This title is very long so should force the alert to add 'title-above'" message="This alert does not have 'title-above' added as a prop but it's forced to add it due to the length of the title."></ic-alert>`,
+    snippet: `<ic-alert heading="I love coffee so much this title could be paragraphs long" message="Coffee is a brewed drink prepared from roasted coffee beans, the seeds of berries from certain flowering plants in the Coffea genus. From the coffee fruit, the seeds are separated to produce a stable, raw product: unroasted green coffee. The seeds are then roasted, a process which transforms them into a consumable product: roasted coffee, which is ground into fine particles that are typically steeped in hot
+water before being filtered out, producing a cup of coffee."></ic-alert>`,
   },
   {
     language: "React",
-    snippet: `<IcAlert heading="This title is very long so should force the alert to add 'title-above'" message="This alert does not have 'title-above' added as a prop but it's forced to add it due to the length of the title." />`,
+    snippet: `<IcAlert heading="I love coffee so much this title could be paragraphs long" message="Coffee is a brewed drink prepared from roasted coffee beans, the seeds of berries from certain flowering plants in the Coffea genus. From the coffee fruit, the seeds are separated to produce a stable, raw product: unroasted green coffee. The seeds are then roasted, a process which transforms them into a consumable product: roasted coffee, which is ground into fine particles that are typically steeped in hot
+water before being filtered out, producing a cup of coffee." />`,
   },
 ];
 
 <ComponentPreview snippets={snippetsResponsiveTitle}>
   <IcAlert
-    heading="This title is very long so should force the alert to add 'title-above'"
-    message="This alert does not have 'title-above' added as a prop but it's forced to add it due to the length of the title."
+    heading="I love coffee so much this title could be paragraphs long"
+    message="Coffee is a brewed drink prepared from roasted coffee beans, the seeds of berries from certain flowering plants in the Coffea genus. From the coffee fruit, the seeds are separated to produce a stable, raw product: unroasted green coffee. The seeds are then roasted, a process which transforms them into a consumable product: roasted coffee, which is ground into fine particles that are typically steeped in hot
```

Comment:
This requires quite a lot of horizontal scrolling, would it be possible to shorten it a bit?